Makefile.am

```diff
@@ -82,103 +82,116 @@ CLEANFILES += $(nodist_pkgconfig_DATA)
 	        s,[@]libdir[@],$(libdir),g; \
 	        s,[@]includedir[@],$(includedir),g;" $^ > $@
 
+# Base TSS2 headers
+tss2dir = $(includedir)/tss2
+tss2_HEADERS = \
+    $(srcdir)/src/include/tss2/tss2_common.h \
+    $(srcdir)/src/include/tss2/tss2_tcti.h \
+    $(srcdir)/src/include/tss2/tss2_tpm2_types.h \
+    $(srcdir)/src/include/tss2/tpm20.h
```

Comment:
+1.
The path $(srcdir)/src/include/tss2 is not what is stated in the commit statement: "This commit updates our source layout so that all headers that are
installed on the platform are in $(srcdir)/include/tss2. When installed
these will end up at $(includedir)/tss2." I think we want $(srcdir)/include/tss2/